site/content/blogs/release-v122.en.md

```diff
@@ -35,6 +35,58 @@ Copilot v1.22 brings several new features and improvements:
 ## IAM Role Permissions Boundary
 
 ## FIFO SNS/SQS
+SNS FIFO Topics and SQS FIFO Queues for your FIFO publish subscribe architecture is now available using Copilot.
+
+####You can configure your manifest to have SNS FIFO Topics enabled for your services. 
+
+You can specify `fifo: true` in your environment manifest under `publish.topics` as shown below and Copilot will create FIFO SNS Topic for you with the default settings given by the CFN.
+
+```yaml
+publish:
+  topics:
+    - name: mytopic
+      fifo: true
+```
+
+Alternatively, you can also specify the advanced FIFO SNS Topic configuration as:
```

Comment:
Shall we briefly explain what `content_based_deduplication` does?

Reply:
I thought of explaining that but then we have 3 more new fields and we need to explain everything in that case. And we are anyway explaining all the fields in the Copilot docs, so they can take a look at there for more information.